### DISCUSSION TITLE
not able to login with test account after running docker

### DISCUSSION THREAD

**Discussion by hoangytm** [featbit/featbit#580] [Q&A] [answered]
git clone https://github.com/featbit/featbit
cd featbit
docker compose up -d
I am following the above and after that docker running without any error.
however, login with the test account test@featbit.com on UI shows the message "Error occurred, please contact the support.". what should I do next to fix this error?

**Comment by deleteLater** [comment]
Hi @hoangytm, I cannot reproduce your problem. Could you

1. Check if all services started
2. Check if there are error logs in the Api service

![image](https://github.com/featbit/featbit/assets/34052208/b3d6b028-3317-41a5-be73-fb1d86e524d1)

**Reply by hoangytm** [reply]
Hi @deleteLater , this is the detail of what I am doing
all the containers are running
![image](https://github.com/featbit/featbit/assets/51194076/9d2c7780-56cf-4a3f-a590-87de81eb3855)
login http://localhost:8081/en/login
![image](https://github.com/featbit/featbit/assets/51194076/f51a3464-2ccb-4e64-b786-4a05810d27a5)
log server
![image](https://github.com/featbit/featbit/assets/51194076/71ec8670-491d-4f2f-b755-875b3d310553)

**Reply by cosmic-flood** [reply]
@hoangytm Could you please open the 'Inspect' window in your browser, switch to the 'Network' tab, and observe what happens after clicking the 'Continue' button? (please make screenshots with more details for us) 

Are you attempting to connect to your Docker-hosted service from a remote machine? In this scenario, the 'localhost' URL will not be accessible. You need to change the UI's URL to enable remote access. (check this [tutorial](https://docs.featbit.co/installation/faq#how-to-make-featbit-portal-accessible-publicly))

![image](https://github.com/featbit/featbit/assets/68597908/1a229552-f953-40f4-a9a0-1a800af99256)
![image](https://github.com/featbit/featbit/assets/68597908/8acff857-989c-4c7d-80f4-3942d3432283)

**Reply by hoangytm** [reply]
@cosmic-flood , some error from console log
![image](https://github.com/featbit/featbit/assets/51194076/b29ea564-2c8f-48c3-b6d4-90d57cebc50e)
![image](https://github.com/featbit/featbit/assets/51194076/c804d935-4acc-494d-8172-af32608eaaec)
![image](https://github.com/featbit/featbit/assets/51194076/e90c51ab-fecb-4ffc-8d52-ef38ed3b6b2b)
and the response tab is empty
I am running docker in my local pc

**Reply by cosmic-flood** [reply]
@hoangytm 

It seems to be a CORS policy issue. Could you try using a different browser to see what happens? If you've been using Chrome, for instance, you might try Edge, or vice versa.

Can you also show more error detail for two "has-multiple-workspaces" requests? （response, headers, etc.）
Can you also try clear your cookie and application cache for "localhost:8081"?

**Reply by hoangytm** [reply]
@cosmic-flood , the issue still occurs on Edge and Firefox, cache and cookie were cleared as well
![image](https://github.com/featbit/featbit/assets/51194076/50038e22-34b9-4eed-a15e-db6dc56b8f04)

**Reply by deleteLater** [reply]
The first request (Method: OPTIONS) is the preflight request, we need to figure out why it fails.

Can you try the following command in PowerShell (or if you are using bash, replace the `` ` ``with `\`)

```powershell
curl -v 'http://localhost:5000/api/v1/user/has-multiple-workspaces' `
  -X 'OPTIONS' `
  -H 'Access-Control-Request-Headers: authorization,content-type,organization,workspace' `
  -H 'Access-Control-Request-Method: POST' `
  -H 'Origin: http://localhost:8081' 
```

A success response should like this:

![image](https://github.com/featbit/featbit/assets/34052208/1d41b2a4-fb6f-4f86-99a1-2f9d0ed90e82)

**Reply by deleteLater** [reply]
Also try this

```shell
curl -X POST http://localhost:5000/api/v1/user/has-multiple-workspaces `
-H 'Content-Type: application/json' `
-d '{
       "email": "test@featbit.com"
    }'
```

**Reply by cosmic-flood** [reply]
Please also check if you have an internet proxy (such as a VPN) enabled, as it can also cause this issue.

**Reply by hoangytm** [reply]
@deleteLater, my results 
![image](https://github.com/featbit/featbit/assets/51194076/167559ef-76b6-4fb0-bc12-f13cc1d45440)
![image](https://github.com/featbit/featbit/assets/51194076/57c62693-943b-4926-a451-639453181f4c)

**Reply by hoangytm** [reply]
@cosmic-flood , I've already turned off VPN.  docker is running on my laptop, I don't think the issue relates internet or network

**Reply by deleteLater** [reply]
Ok, the server returns 404, maybe you use old Api image. Can you try to delete all featbit image then rerun `docker compose up -d`?

**Reply by hoangytm** [reply]
@deleteLater I have just downloaded it, it is the latest version. Anyway, I deleted the image, and container and ran docker-compose again. the issue still happens
![image](https://github.com/featbit/featbit/assets/51194076/c0bc714f-f9ad-4430-ad4d-c6dfdd58da76)

**Reply by deleteLater** [reply]
How about `curl http://localhost:5000/health/liveness`

**Reply by hoangytm** [reply]
it returns 404 as well 
![image](https://github.com/featbit/featbit/assets/51194076/6b7b2bac-9f5e-4c93-8417-c55c531c6968)

**Reply by deleteLater** [reply, marked as answer]
Very strange, I can sure that the Api service provides this endpoint. Maybe you need to check your network and figure out why it returns `404`.

**Reply by hoangytm** [reply]
Maybe, my laptop conflicts with port 5000. I changed the default public port 5000 to 5001, and it works well.  
Thank @deleteLater  and @cosmic-flood  for your supports

**Reply by cosmos-explorer** [reply]
@hoangytm just a question, are you using macbook?

**Reply by hoangytm** [reply]
no, I am using dell with window 10
